src/redux/store.js

```diff
@@ -0,0 +1,4 @@
+import { createStore } from "redux";
+import solrReducer from "./solr-reducer";
+
+export const store = createStore(solrReducer);
```

Comment:
Moving all redux stuff to the redux folder in case we need to add more reducers when the application grows.